automotive/src/main/java/au/com/shiftyjelly/pocketcasts/AutomotiveSettingsActivity.kt

```diff
@@ -127,6 +128,7 @@ class AutomotiveSettingsActivity : AppCompatActivity(), FragmentHostListener {
         source: EpisodeViewSource,
         podcastUuid: String?,
         forceDark: Boolean,
+        timestampInSecs: Duration?,
```

Comment:
This can now be called `timestamp`.

Reply:
In https://github.com/Automattic/pocket-casts-android/pull/2022/commits/e860d22e43a9fe12bd274f30335b282ff229872f